### PR TITLE
Add Pulumi Kubernetes Operator 1.0 Blog Post

### PR DESCRIPTION
**Pull request by lukehoban**

### CODE REVIEW

File: themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+date: "2021-10-12"
+title: "Pulumi Kubernetes Operator 1.0"
+authors: ["luke-hoban", "vivek-lakshmanan"]
+tags: ["Kubernetes", "Continuous-Delivery", "operators"]
+meta_desc: "Pulumi Kubernetes Operator 1.0: GitOps, Automation API, State Backends, Safe Upgrades and Metrics"
+meta_image: operator.png
+---
+
+Last year we released the [Pulumi Kubernetes Operator]({{< relref "/blog/pulumi-kubernetes-operator" >}}), a new cloud-native way to manage and deploy cloud infrastructure using Pulumi from within your Kubernetes environment. Since then, we’ve worked with many Pulumi users who have adopted the Pulumi Kubernetes Operator at increasingly larger scales and for a wide variety of use cases. Today, we’re excited to make the [1.0 release](TBD) of the Pulumi Kubernetes Operator available.
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md:10]: @viveklak Could you add or suggest the best target link for "1.0 release" here?

File: themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+This included the use of several capabilities from the Kubernetes Operator SDK, including [Leader Election](https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#leader-with-lease), [Graceful Shutdown](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Options) and more.
+
+### Prometheus Metrics
+
+The Pulumi Kubernetes Operator automatically manages a lot of details of Pulumi deployments, and as a result, it’s critical to be able to get insight into the operation of the Operator. Previously, this was available either via operator logs, or via the Pulumi Console. As part of the 1.0 release, the Operator is also now exporting Prometheus-compatible metrics for `stacks_active` and `stacks_failing`. These can be used to drive reporting and other workflows inside the Kubernetes cluster.
+
+<!-- TODO: Screenshot or curl output of metrics? -->
+
+<!-- TODO: ### Documentation -->
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md:99]: @viveklak Can you add these two sections, or share the pointers to the new docs I can add in here once they are available?

File: themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+This includes capabilities like [Leader Election](https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#leader-with-lease), [Graceful Shutdown](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Options) and more.
+
+### Prometheus Metrics
+
+The Pulumi Kubernetes Operator automatically manages a lot of details of Pulumi deployments, and as a result, it’s critical to be able to get insight into the operation of the Operator. Previously, this was available either via operator logs, or via the Pulumi Console. As part of the 1.0 release, the Operator is also now exporting Prometheus-compatible metrics for `stacks_active` and `stacks_failing`. These can be used to drive reporting and other workflows inside the Kubernetes cluster.
+
+<!-- TODO: Screenshot or curl output of metrics? -->
+
+<!-- TODO: ### Documentation -->
```
Comment (viveklak) [inline themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md:99]:
```suggestion
The Pulumi Kubernetes Operator automatically manages a lot of details of Pulumi deployments, and as a result, it’s critical to be able to get insight into the operation of the Operator. Previously, this was available either via operator logs, or via the Pulumi Console. As part of the 1.0 release, the Operator is also now exporting [Prometheus-compatible metrics](https://github.com/pulumi/pulumi-kubernetes-operator/pull/217#issue-1021521569) for `stacks_active` and `stacks_failing`. These can be used to drive reporting and other workflows inside the Kubernetes cluster.

<!-- TODO: ### Add metrics Documentation once https://github.com/pulumi/pulumi-kubernetes-operator/issues/218 is addressed. -->
```

There seem to be some issues with the service monitors configured by default by the operator framework for scraping metrics. I am tracking better out-of-the-box prometheus integration in https://github.com/pulumi/pulumi-kubernetes-operator/issues/218. Will follow-up with links separately when I fix that.

File: themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+# TYPE stacks_active gauge
+stacks_active 2
+# HELP stacks_failing Number of stacks currently registered where the last reconcile failed
+# TYPE stacks_failing gauge
+stacks_failing{name="stack-test-aws-s3-commit-change-5wuczx",namespace="default"} 1
+```
+
+### Documentation
+
+We've added new documentation and enhanced existing documentatin to ensure it's easy to get started with the Pulumi Kubernetes Operator.
```
Comment (lblackstone) [inline themes/default/content/blog/pulumi-kubernetes-operator-1-0/index.md:114]:
```suggestion
We've added new documentation and enhanced existing documentation to ensure it's easy to get started with the Pulumi Kubernetes Operator.
```

**Pull request merged**